packages/snowpack/src/commands/build.ts

```diff
@@ -7,11 +7,11 @@ import * as colors from 'kleur/colors';
 import mkdirp from 'mkdirp';
 import path from 'path';
 import rimraf from 'rimraf';
-import {SnowpackSourceFile, removeLeadingSlash} from '../config';
+import {removeLeadingSlash, SnowpackSourceFile, removeTrailingSlash} from '../config';
```

Comment:
You alphabetized one, but not the other? 😉

Reply:
lol now I'm scared to make a change given all the commits on top of one another. Will be sure to re-organize in the final PR